roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -1,6 +1,4 @@
 ---
-run_gitinfos: false
```

Comment:
Not really intended but this variable is unused.. if need be I can cleanup the PR and remove this change